main_test.go

```diff
@@ -713,3 +716,37 @@ func Test_runScorecardSettings(t *testing.T) {
 		})
 	}
 }
+
+func Test_signScorecardResult(t *testing.T) {
+	t.Parallel()
+	// Sign example scorecard results file.
+	scorecardResultsFile := "./testdata/scorecard-results-example.sarif"
+	err := signScorecardResult(scorecardResultsFile)
+	if err != nil {
+		t.Errorf("signScorecardResult() error, %v", err)
+		return
+	}
+
+	// Verify that the signature was created and uploaded to the Rekor tlog by looking up the payload.
+	ctx := context.Background()
+	rekorClient, err := rekor.NewClient("https://rekor.sigstore.dev")
+	if err != nil {
+		t.Errorf("signScorecardResult() error getting Rekor client, %v", err)
+		return
+	}
+	scorecardResultData, err := ioutil.ReadFile("./testdata/scorecard-results-example.sarif")
+	if err != nil {
+		t.Errorf("signScorecardResult() error reading scorecard result file, %v", err)
+		return
+	}
+	uuids, err := cosign.FindTLogEntriesByPayload(ctx, rekorClient, scorecardResultData)
```

Comment:
This is an FYI for later: `FindTLogEntriesByPayload` uses the search index on top of rekor to find the payload. it does NOT verify the UUIDs are actually in the log! Probably fine for a unit test, but the API service validating will need to do additional verification to check if the UUID is part of the tlog.

(see https://github.com/sigstore/cosign/blob/18d2ce0b458018951f7356db911467a427a8dffe/pkg/cosign/tlog.go#L225)

Reply:
`verifyTlogEntry` is private so I can't import it. Is there any workaround?